src/System.Text.Primitives/System/Text/Parsing/InvariantUtf8/InvariantUtf8_UInt32.cs

```diff
@@ -11,20 +11,44 @@ public static partial class PrimitiveParser
     {
         public static partial class InvariantUtf8
         {
-            static int UINT_OVERFLOW_LENGTH = 10;
+            #region static
+            private static readonly int UINT_OVERFLOW_LENGTH = 10;
+            private static readonly int UINT_OVERFLOW_LENGTH_HEX = 8;
+
+            static readonly int[] hexLookup =
```

Comment:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md say to prefix static fields with `s_`.